### PR TITLE
Add send SMS docs

### PR DESCRIPTION
**Pull request by jorunfa**

### CODE REVIEW

File: docs/send-sms-explanation-outgoing-vs-incoming.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+title: SMS direction OUTGOING vs INCOMING
+topic: Send SMS
+type: explanation
+---
+
+# SMS direction OUTGOING vs INCOMING
+
+You can send an SMS with a `Direction` either as `OUTGOING` or `INCOMING`.
+
+A traditional SMS sent from one of your subscribers to another subscriber passes through the system two times. One time as `OUTGOING` and once as `INCOMING`. With the correct permissions it is possible to omit the outgoing SMS if you are sending to one of your subscribers. What to use depends on what kind of scenario you're facing.
```
Comment (didier-wg2) [inline docs/send-sms-explanation-outgoing-vs-incoming.md]: I think it only passes through the system twice if both the sender and receiver are subscribers of us.
Outgoing means that the message is going from one of our subscribers.
Incoming means the message is going to one of our subscribers.

File: docs/send-sms-how-to.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+## Overview
+
+To send an SMS you will need to:
+* [Get credentials from Console](https://console.wgtwo.com/api-keys-redirect)
+* Write code that that targets a sender, receiver, and with some content
+
+## Prerequisites
+
+### Token/credentials
+* You will need [credentials from Console](https://console.wgtwo.com/api-keys-redirect) to list voicemails for users on your platform.
```
Comment (didier-wg2) [inline docs/send-sms-how-to.md]: `voicemails` copy pasta

File: docs/send-sms-explanation-outgoing-vs-incoming.md (hunk 1 above)
Comment (jorunfa) [reply docs/send-sms-explanation-outgoing-vs-incoming.md]: Good point, I'll change it.

File: docs/send-sms-how-to.md (hunk 2 above)
Comment (jorunfa) [reply docs/send-sms-how-to.md]: Thanks.

File: docs/send-sms-explanation-outgoing-vs-incoming.md (hunk 1 above)
Comment (jorunfa) [reply docs/send-sms-explanation-outgoing-vs-incoming.md]: Changed. Better?

File: docs/send-sms-how-to.md (hunk 2 above)
Comment (jorunfa) [reply docs/send-sms-how-to.md]: Changed. Better?

File: docs/send-sms-explanation-outgoing-vs-incoming.md (hunk 1 above)
Comment (jorunfa) [reply docs/send-sms-explanation-outgoing-vs-incoming.md]: Changed. Better?

**Pull request merged**